client-java/controller-api/src/main/java/org/evomaster/client/java/controller/api/EMTestUtils.java

```diff
@@ -37,7 +37,7 @@ public static String resolveLocation(String locationHeader, String expectedTempl
         try{
             locationURI = URI.create(locationHeader);
         } catch (Exception e){
-            return locationHeader;
+            return expectedTemplate;
```

Comment:
Hi! thanks for this PR! ;)
But I m not sure I do agree on this specific change.
If the Location header is wrong (ie it is not a valid URI), what would be the point to return the `expectedTemplate` here?
The idea was that, if a location header is wrong, to still have it in the generated tests, so a user can see that the SUT is doing something wrong. 
But, still, the EvoMaster process should not crash

Reply:
Oh, you are right, I misunderstood this method. I'll undo this change, as with only changing the method `getResourceId()`, EvoMaster doesn't crash.